firstvoices/backend/search/query_builder.py

```diff
@@ -20,6 +21,7 @@ def get_search_object(indices):
 
 
 def get_search_query(
+    user=None,
```

Comment:
Possible security issue here? I had made user an optional paramater so that the other tests pass without major changes, but if an attacker were to spoof that there is no user then view permissions would not be active. However, I'm not sure if that's possible.

Reply:
Better default would be AnonymousUser()